src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -69,7 +69,19 @@
    :ruby-load-path           [schema/Str]
    :gem-path                 schema/Str
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   ;; This would be list of compile master node names (FQDNs) including MoM
+   ;; It would either be provisioned by PE or created & maintained by user (input configuration)
+   :compile-masters          [schema/Str]    
```

Comment:
I'd maybe change this to be "infra-certs" or "infra-certnames" to keep consistent with our usage of "infra" elsewhere (ie it's a "infra-crl" not a "compiler-masters-crl".

Reply:
It depends on what infrastructure constitutes like if it also includes things like puppetdb and console nodes in case of an split install then it would be misleading. I just wanted to be specific to only mean compile masters. But no heartburn if you prefer it like that. 
By that token then infra-crl is misleading as well...will change it I guess (in a bit).